addons/cluster-autoscaler/cluster-autoscaler.sh

```diff
@@ -91,7 +91,16 @@ aws iam attach-role-policy --policy-arn $ASG_POLICY_ARN --role-name $IAM_ROLE
 printf " ✅ \n"
 
 addon=cluster-autoscaler.yml
-wget -O ${addon} https://raw.githubusercontent.com/kubernetes/kops/master/addons/cluster-autoscaler/v1.8.0.yaml
+manifest_url=https://raw.githubusercontent.com/kubernetes/kops/master/addons/cluster-autoscaler/v1.8.0.yaml
+
+if which -s wget; then
```

Comment:
Unfortunately I don't think `-s` is supported on linux. 

You could use this trick: https://github.com/kubernetes/kubernetes/blob/6696ae39ee7df5323e843d93c6146e51d297a299/cluster/get-kube-local.sh#L132